src/BuildingRegistry.Api.CrabImport/Infrastructure/Modules/ApiModule.cs

```diff
@@ -62,13 +62,13 @@ protected override void Load(ContainerBuilder containerBuilder)
                     Schema.Import,
                     _loggerFactory));
 
-            containerBuilder
-                .RegisterType<IdempotentCommandHandlerModule>()
-                .AsSelf();
+            //containerBuilder
+            //    .RegisterType<IdempotentCommandHandlerModule>()
+            //    .AsSelf();
 
-            containerBuilder
-                .RegisterType<IdempotentCommandHandlerModuleProcessor>()
-                .As<IIdempotentCommandHandlerModuleProcessor>();
+            //containerBuilder
```

Comment:
Why are these lines commented out?

Reply:
Because the two registrations were redundant, see under the commented block. Iv removed the commented block now.